test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -74,18 +76,24 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, Label(string(lab
 		profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
 		Expect(err).ToNot(HaveOccurred())
 
-		performanceMCP, err = mcps.GetByProfile(profile)
-		Expect(err).ToNot(HaveOccurred())
+		if !hypershift.IsHypershiftCluster() {
```

Comment:
I would suggest to add
`GetPoolName` function that would handle the case for both OCP and HCP